forge-gui/res/cardsfolder/upcoming/horned_loch_whale_lagoon_breach.txt

```diff
@@ -0,0 +1,18 @@
+Name:Horned Loch-Whale
+ManaCost:4 U U
+Types:Creature Whale
+PT:6/6
+K:Flash
+K:Ward:2
+K:ETBReplacement:Other:LandTapped
+SVar:LandTapped:DB$ Tap | Defined$ Self | ETB$ True | ConditionOpponentTurn$ True | SpellDescription$ CARDNAME enters the battlefield tapped unless it's your turn.
```

Comment:
probably need some PlayerCount check for your ally turns

Reply:
I  probably would add a more generic Turn Check for this

Reply:
@Simisays do this: `ConditionPlayerTurn$ False`
https://github.com/Card-Forge/forge/blob/master/forge-game/src/main/java/forge/game/spellability/SpellAbilityCondition.java#L335

Reply:
thanks, working now